pkg/alertmanager/types.go

```diff
@@ -473,12 +473,13 @@ type rocketchatAttachmentAction struct {
 }
 
 type rocketChatConfig struct {
-	HTTPConfig  *httpClientConfig `yaml:"http_config,omitempty"`
-	APIURL      string            `yaml:"api_url,omitempty"`
-	TokenID     *string           `yaml:"token_id,omitempty"`
-	TokenIDFile string            `yaml:"token_id_file,omitempty"`
-	Token       *string           `yaml:"token,omitempty"`
-	TokenFile   string            `yaml:"token_file,omitempty"`
+	VSendResolved *bool             `yaml:"send_resolved,omitempty"`
```

Comment:
In here actually needs to have a similar structure to the others, with the json and yaml structure, for example https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/alertmanager/types.go#L129